### DISCUSSION TITLE
Slowdown S-Turn

### DISCUSSION THREAD

**Discussion by gnbo** [voxeltycoon/issues#672] [General]
**Describe the suggestion:**
The trains slow down when driving through curves with 90 ° or more. That's a great thing. However, they also brake on an S-bend with two 45 ° bends. In this way, they reduce the speed with a simple track change as with a tight 90 ° curve. I would like there to be no (or little) speed reduction. 

**Current behavior:**
Build a Railroad with like this and send off a train:
(This train travels 60km/h in a straight line, only 40km/h in 90 ° and S-bends.) 

![grafik](https://user-images.githubusercontent.com/23659904/116779178-f8adcc80-aa74-11eb-857f-6e4a4213dbc5.png)

**Comment by victorleblais** [comment]
Hi, this is not a bug, so I changed your post to a discussion, which are there to make suggestions for future changes and features.

**Comment by tycode** [comment]
Ideally larger radius curves would be lovely.

Those are probably very difficult to add though, so it would be good if the game only applied a speed limit if the train passed through 2 curves in the same direction consecutively (both left or both right), and didn't apply the speed limit if the curves are in opposite directions (which happens in an S-bend like this).
